core/src/components/popover/popover.ios.scss

```diff
@@ -75,17 +75,13 @@
   transform: rotate(180deg);
 }
 
-:host(.popover-side-start) .popover-arrow {
-  @include rtl() {
-    transform: rotate(-90deg);
-  }
+:host(.popover-side-start.popover-ltr) .popover-arrow,
```

Comment:
Checking my understanding: We removed the `@include rtl` mixin usage in favor of explicit RTL/LTR classes to fix support for firefox/safari, correct?

Reply:
Yes and no. When I removed `host-context` then the arrow was no longer displaying correctly for Chrome. Since popover is [using the shadow DOM](https://github.com/ionic-team/ionic-framework/pull/27203/files#diff-002321860b0f9ec9fd960b5274d8a90c3d7be3246b03be0510e154562ea104c3R161-R162), it doesn't know what direction the document it is so classes was my solution. This ended fixing it for Firefox and Safari.